bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -222,6 +222,15 @@ private function addGeneralNode(ArrayNodeDefinition $rootNode): void
                     ->defaultValue(['en', 'de', 'fr'])
                     ->prototype('scalar')->end()
                 ->end()
+                ->arrayNode('required_languages')
+                    ->info('String or array format are supported.')
+                    ->beforeNormalization()
+                    ->ifString()
+                        ->then(fn ($v) => explode(',', $v))
+                    ->end()
+                    ->defaultValue(['en', 'de', 'fr'])
```

Comment:
I think the default value should be empty and a empty array should make all valid languages required. Otherwise we have a behavior change after an update, right?

Reply:
Very good point, yes. Maybe it would already be enough to just return `Tool::getValidLanguages()` from within `Tool::getRequiredLanguages()` instead of `return []`

Reply:
Good Point. Thought setting the same value as the valid languages would be enough to not have problems after the update, but you are right this could have caused issues. I liked the idea to just use the `Tool::getValidLangauages()`  function as fallback, to have the same languages and therefore make all required. Having an empty array as all would have made everything else slightly different, so I thought this also would be the better solution.